.env.example

```diff
@@ -16,6 +16,7 @@ TRANSMORPHER_DEV_MODE=false
 TRANSMORPHER_STORE_DERIVATIVES=true
 TRANSMORPHER_DISK_ORIGINALS=localOriginals
 TRANSMORPHER_DISK_IMAGE_DERIVATIVES=localImageDerivatives
+TRANSMORPHER_DISK_PDF_DERIVATIVES=localPdfDerivatives
```

Comment:
we might want to call this the document disk, just in case we ever support a different format

Reply:
done